### ISSUE TITLE
Scenario Bundles: Same scenarios in different studies cannot compare across studies

### ISSUE THREAD

**Issue by han-f** [OpenEnergyPlatform/oeplatform#1411]
Not sure whether this is a bug or a feature request ;-)

There are WEM and WAM scenarios in two studies, which cover however different regions:

- Consolidated and Harmonised GHG Emission Projections from European Legislation covers all EU Member States
- Rahmendaten für dem Projektionsbericht 2023 covers only Germany  

When I select WEM and WAM to compare, I only receive the information for the first study, while I expected to also see the WEM and WAM for the study covering only Germany.  The definition is exactly the same (albeit I used a bit of a different text to describe them in the edit interface), so as a user I would be interested to also be able to see all the studies who considered the WEM and WAM scenario explicitly.

May this be solved by also adding study name to the cards, and display an individual card per study that uses the scenario?

cc @l-emele

**Comment by jh-RLI** [comment]
Good point ... I'm not really sure about that at this point. It seems like we assumed that a particular scenario, even if used in different studies, remains the same. 

I like the idea of adding the related studies to each of the cards.
@adelmemariani

**Comment by l-emele** [comment]
> As @jh-RLI mentioned, our assumption was that scenarios are unique (considering the acronyms).

At some point, we discussed that only the combination study + scenario has to be unique.

**Comment by jh-RLI** [comment]
The problem has already been fixed. You can now select and compare a separate scenario for each scenario bundle. Please open again if the problem persists.
